pkg/operator/ceph/nfs/controller.go

```diff
@@ -85,6 +85,11 @@ func (c *CephNFSController) StartWatch(namespace string, stopCh chan struct{}) e
 }
 
 func (c *CephNFSController) onAdd(obj interface{}) {
+	if c.clusterSpec.External.Enable && !c.clusterSpec.External.RunStatelessWorkload {
+		logger.Warningf("Creating nfs for an external ceph cluster is disabled (RunStatelessWorkload is false)")
```

Comment:
Nit: from the user-facing side of things, this is `runStatelessWorkload` not capitalized.